apps/webservice/src/app/[workspaceSlug]/(app)/_components/environment-policy-drawer/PolicyFormSchema.ts

```
@@ -0,0 +1,19 @@
+import type { UseFormReturn } from "react-hook-form";
+import ms from "ms";
+import { z } from "zod";
+
+import * as SCHEMA from "@ctrlplane/db/schema";
+
+const isValidDuration = (str: string) => !isNaN(ms(str));
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)